rasa_core/policies/two_stage_fallback.py

```diff
@@ -41,9 +41,11 @@ class TwoStageFallbackPolicy(FallbackPolicy):
     """
 
     def __init__(self,
+                 priority: int = 3,
                  nlu_threshold: float = 0.3,
                  core_threshold: float = 0.3,
-                 fallback_core_action_name: Text = ACTION_DEFAULT_FALLBACK_NAME,
+                 fallback_core_action_name:
```

Comment:
do we need this line break?

Reply:
my editor was yelling at me because it was 80 instead of 79, not sure what our default is, because it probably will pass at 80 (because it obviously passed before)